tools/spec-gen-sdk/src/automation/reportStatus.ts

```diff
@@ -155,15 +155,14 @@ export const saveFilteredLog = (context: WorkflowContext) => {
     notEnabled: 'Warning'
   } as const;
   const type = statusMap[context.status];
-  const filteredResultData = [
-    {
-      type: 'Markdown',
-      mode: 'replace',
-      level: type,
-      message: commentBody,
-      time: new Date()
-    } as MessageRecord
-  ].concat(context.extraResultRecords);
```

Comment:
Where does `context.extraResultRecords` go? Don't we need to include that in the log?

Reply:
It is not being used in the code, and it is always an empty array. Removed